### ISSUE TITLE
Color Picker - Element gradient display - Scaffolding 

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#403]
## Feature Description

This feature is split from #263. 
This deals with building the SCAFFOLDING for each of the gradient types. 
See #1387 #1388 #1389 for the implementation of the gradient types. 

When an element is selected and it has a gradient applied somehow (text color, background color, shape color) or when nothing or the background element is selected and it has a gradient applied (page background color), display gradient stops and shape **on** the selected element or page background respectively.

[figma](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=3138%3A103)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

* **Design**

Change the direction **and** size of a linear, conic or radial gradient (only makes sense if radial is non-circular of course):
![Screen Shot 2020-04-14 at 11 58 14 AM](https://user-images.githubusercontent.com/637548/79246701-8c4f3380-7e47-11ea-91e5-936d22b7d3ae.png)

Change the shape (size) of a radial gradient:
![Screen Shot 2020-04-14 at 11 58 20 AM](https://user-images.githubusercontent.com/637548/79246705-8d806080-7e47-11ea-9ba4-e817892881f4.png)

Similarly, there will be a way to move the center of a radial or conical gradient (not designed).

* **Epic**
* **Related issues / PRs:**

## Acceptance Criteria

1. The scaffolding is built for the linear gradient #1387 
2. The scaffolding is built for the radial gradient #1388
3. The scaffolding is built for the conic gradient #1389

**Comment by samitron7** [comment]
Removed ux label as UI is on Figma under the color section [https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=3138%3A103](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=3138%3A103)

**Comment by miina** [comment]
@barklund @jhtin If I understand correctly then #1387 #1388 and #1389 are all blocked by this issue, could you confirm just in case before I mark those issues blocked.

**Comment by bmattb** [comment]
@o-fernandez - we  _think_ that  this is flagged on line 22 in the roadmap document, hence the 1.2 assignment, please confirm

**Comment by o-fernandez** [comment]
> @o-fernandez - we _think_ that this is flagged on line 22 in the roadmap document, hence the 1.2 assignment, please confirm

@bmattb this was incorrectly tagged to line 22 of the roadmap **Fixes: Color picker (partially implemented, not to spec)** but after reviewing the story points required (20+ SP!) and discussing with @samitron7, Sam clarified that what's really important is not the gradient display work for this. 

Sam is going to **create a new ticket** for what's really needed in terms of color picker fixes and we can then attach that to 1.2 line 22. It should be much smaller in scope that this ticket.

**Comment by samitron7** [comment]
@bmattb Let's close out this ticket, [#1387](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/1387) and [#1388](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/1388). These 3 tickets are found inside this [Epic $4524](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/4524), which we can close out also. In short, this is an advance designer feature that is a nice to have but can be part of our wish list.

In place of these tickets, I created an Epic [#5111](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/5111) for color picker. Can we prioritize performant issues first (#4463 and #1901). The color picker seems broken to me if its not responsive.
